app_server/app/models/Config.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Edit to your local mysql database's credentials
+ */
+
+var config = {};
+
+config.name = '';
+config.username = '';
+config.password = '';
+config.host = 'localhost';
+config.dialect = 'mysql';
```

Comment:
Shifted this config template file over to app_server/config. I will add it into git ignore after everyone has a copy of this.